### ISSUE TITLE
Error: Failed to connect to network share: 1203

### ISSUE THREAD

**Issue by jacksomness** [Kudayasu/Artifice#3]
I tried the exploit on different versions and used both the hostname and the IP address of my dev kit and I still get this error. After the error, the program halts at Setting up SSH.

**Comment by Kudayasu** [comment]
Few things you can try:

- Ensure `net use` in a cmd window returns an `OK` status for your console, i.e. `\\192.168.1.123\DevelopmentFiles`
- Ensure you don't have any saved **Windows Credentials** for your console hostname or IP in [Credential Manager](https://support.microsoft.com/en-us/windows/accessing-credential-manager-1b5c916a-6a16-889f-8581-fc16e8165ac0)
- Manually clear any saved connections via `net use * /d`
- You're able to manually access your dev scratch in explorer at the path listed
- You're able to login manually with ssh via [DevToolsUser](https://xboxresearch.com/wiki/setup-dev-mode)
- Console is on the same subnet as your PC _(I doubt this is the issue, but it's worth a guess)_
- Reduce any possible variables for where the program executes, i.e. extract the `publish` folder to your **Desktop** and run it from there.

Otherwise if you've gone through these steps, especially if you're able to manually access the smb share, the next best suggestion I have is starting with a clean slate. Restart your PC, try wired instead of wireless & vice-versa if possible. 

I highly doubt it's a console-side issue, but for the sake of completeness, you could attempt a factory reset if you so wish.

I have yet to encounter error 1203 and haven't been able to reproduce it myself, so let me know what your network setup is, what insider ring you're in _(if any)_, what SKU your console is, etc.

**Comment by jacksomness** [comment]
I tried everything you've listed and I got the same error. I ran Artifice on 2 other computers and it was partially successful, the error is gone now. But now its failing to install art.msix on both computers. It didn't show any error code either. I tried to manually install art.msix and its dependencies but that didn't work. The 2 computers that have this error are fresh installs of windows, one with windows 10 the other with 11. I don't know if I'm missing some software or something. I obviously have NET 8.0 installed and I even tried to install every windows feature that has anything to do with SMB, TCPIP, SSH, UWP, etc. But I'm still getting the error. I'm not an expert in this matter so I'm pretty much stumped. If you have any more suggestions please let me know. Thanks.

**Comment by Kudayasu** [comment]
If you're able to, send a screenshot of where the tool gets stuck with installing the app. Given your mention regarding the lack of any errors, it sounds similar to this [previous issue](https://github.com/Kudayasu/Artifice/issues/1)

Otherwise, with how Artifice is setup to rely on the four-stage sequence of (re)launching the app, manual intervention will do more harm than good unfortunately.

**Comment by jacksomness** [comment]
Sorry for the late reply, but after a bit of troubleshooting and code reading, I found the solution. I was testing Artifice in my downloads folder and on my desktop and it wasn't working at first, but once I placed the publish folder in the root of my C:\ drive, it worked. I have no clue why, but now if anyone else gets one of these 2 errors I got, just point them here.

I also installed .NET 4.8 via Windows Features before I moved the folder into my C:\ drive and it didn't have an effect then, but maybe putting it in my C:\ drive allowed the WDP driver to easily find .NET 4.8 in PATH. I'm mentioning this since I noticed the config file for XboxWdpDriver.exe states its .NET runtime version 4.8, but I don't really know if that meant anything.

I also tested running the install command manually through cmd and even though I was running the same command from Task StepTwo(), in MainWindow.xaml.cs, with the same syntax and everything, it was asking me to provide a subop argument. But after moving the publish folder into C:\, the installation completes with no errors, so maybe .NET 4.8 is needed and the placement of the publish folder depends on how .NET 4.8 was installed since the PATH variable might be different, if installed via Windows Features, the official website, etc.
